README.md

```diff
@@ -2153,8 +2153,7 @@ function addAProductToTheDealEventuallyCreatingANewItemCalledADealProduct($optio
 | Parameter | Tags | Description |
 |-----------|------|-------------|
 | id |  ``` Required ```  | ID of the deal |
-| body |  ``` Optional ```  | TODO: Add a parameter description |
-
+| body |  ``` Required ```  | [Body object](https://developers.pipedrive.com/docs/api/v1/Deals#addDealProduct) that has all required body parameters |
 
 
 #### Example Usage

```

Comment:
Maybe it would be user-friendly to update the Example usage section as well: $deals->addAProductToADeal($collect);. Maybe for "backward compatibility" it is needed to keep the current longer version as well (i.e as a comment on the other line).

Reply:
It makes total sense, will prepare an update!